tests/handlers/aws/test_handler.py

```diff
@@ -258,103 +300,127 @@ def test_lambda_handler(self) -> None:
-                with mock.patch("shippers.ElasticsearchShipper._elasticsearch_client", self._es_client_patch):
-
-                    ctx = ContextMock()
-                    event = {
-                        "Records": [
-                            {
-                                "messageId": "9b745861-1171-489c-9748-799ed2a3d9da",
-                                "body": json.dumps(
-                                    {
-                                        "Records": [
-                                            {
-                                                "eventVersion": "2.1",
-                                                "eventSource": "aws:s3",
-                                                "awsRegion": "eu-central-1",
-                                                "eventTime": "2021-09-08T18:34:25.042Z",
-                                                "eventName": "ObjectCreated:Put",
-                                                "s3": {
-                                                    "s3SchemaVersion": "1.0",
-                                                    "configurationId": "test-bucket",
-                                                    "bucket": {
-                                                        "name": "test-bucket",
-                                                        "arn": "arn:aws:s3:::test-bucket",
-                                                    },
-                                                    "object": {
-                                                        "key": f"{filename}",
+                with mock.patch(
+                    "share.secretsmanager._get_aws_sm_client",
+                    lambda region_name: aws_stack.connect_to_service(
+                        "secretsmanager",
+                        endpoint_url=f"http://localhost:{self._LOCALSTACK_HOST_PORT}",
+                        region_name=region_name,
+                    ),
+                ):
+                    with mock.patch("shippers.ElasticsearchShipper._elasticsearch_client", self._es_client_patch):
+
+                        parsed_config_yml: str = f"""
+                        inputs:
+                          - type: "sqs"
+                            id: "{self._source_queue_info["QueueArn"]}"
+                            outputs:
+                              - type: "elasticsearch"
+                                args:
+                                  elasticsearch_url: "http://127.0.0.1:{self._ES_HOST_PORT}"
+                                  username: "{self._ELASTIC_USER}"
+                                  password: "{self._ELASTIC_PASSWORD}"
+                                  dataset: "redis.log"
+                                  namespace: "default"
+                                """
+
+                        assert aws_sm_expander(self._config_yaml) == parsed_config_yml
```

Comment:
we can skip this in the integration test: we already have unit coverage for this.
the integration test should fail if the config is not parsed properly